src/main.cpp

```diff
@@ -981,6 +981,36 @@ bool ContextualCheckTransaction(
             // https://zips.z.cash/zip-0213#specification
             uint256 ovk;
             for (const OutputDescription &output : tx.vShieldedOutput) {
+                bool zip_212_enabled;
+                libzcash::SaplingPaymentAddress zaddr;
+                CAmount value;
+
+            // 5.4.2 EoS height is 2121024.
+            if (nHeight >= 2121200) {
```

Comment:
Oof, while I appreciate the preservation of indentation, I'd rather we just review this ignoring whitespace changes.

Reply:
The negative indentation here is intentional, both to reduce the number of lines changed in this PR, and to do the same in the future PR where the old code is removed.

Reply:
I really strongly disagree that reducing the number of lines changed should take priority over correct indentation. It makes it confusing to read, which is especially bad for consensus code.

Reply:
My desire here was not specifically to reduce the number of lines changed, but to enable `git blame` to work on the new code nicely without the indirection of an unindenting commit. If the old code were going to stick around then I'd absolutely agree that this should be implemented, but conversely if the old code were being removed immediately then we wouldn't be indenting at all. This code falls in between these two edges, and I chose to tie-break towards the latter given the expected lifetime of the old code.

Reply:
Suggested compromise: https://github.com/str4d/zcash/pull/24/files

Reply:
If you do the indentation changes in a separate commit, you can add that commit to [`blame.ignoreRevsFile`](https://www.git-scm.com/docs/git-config#Documentation/git-config.txt-blameignoreRevsFile).